test/e2e/funcs/feature.go

```diff
@@ -219,7 +219,11 @@ func ResourcesHaveConditionWithin(d time.Duration, dir, pattern string, cds ...x
 			_ = fieldpath.Pave(u.Object).GetValueInto("status", &s)
 
 			for _, want := range cds {
-				if !s.GetCondition(want.Type).Equal(want) {
+				got := s.GetCondition(want.Type)
+				if want.Message == "" {
+					got.Message = ""
```

Comment:
Is this intentional? 
I am reading this as "ignore what you get as a message as long as other fields of the conditions match", right?

Reply:
Yes. Messages are for humans, not tests.

Reply:
But we could still test/validate that we emit the correct messages for humans, right? 

I can understand that it could be unfeasible in some cases, so don't have a strong opinion to keep them.

Reply:
Not in e2e. In unit tests yes.